src/app/shared/components/Breadcrumbs/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -231,131 +241,164 @@ const BreadcrumbsItem = (props: BreadcrumbsItemProps): JSX.Element => {
                 'outline-none absolute mt-1 w-56 rounded-md border border-black border-opacity-8 bg-white py-1.5 text-base shadow-subtle-hard'
               }
             >
-              <Menu.Item>
-                {({ active }) => (
-                  <div
-                    onClick={onCreateFolderButtonClicked}
-                    className={`${
-                      active && 'bg-gray-5'
-                    } flex cursor-pointer items-center py-2 px-3 text-gray-80 hover:bg-gray-5`}
-                  >
-                    <FolderSimplePlus size={20} />
-                    <p className="ml-3">{i18n.get('actions.upload.folder')}</p>
-                  </div>
-                )}
-              </Menu.Item>
-              <div className="my-0.5 mx-3 border-t border-gray-10" />
-              {!isBreadcrumbItemShared ? (
-                <Menu.Item>
-                  {({ active }) => (
-                    <div
-                      onClick={onCreateLinkButtonClicked}
-                      className={`${
-                        active && 'bg-gray-5'
-                      } flex cursor-pointer items-center py-2 px-3 text-gray-80 hover:bg-gray-5`}
-                    >
-                      <Link size={20} />
-                      <p className="ml-3">Get Link</p>
-                    </div>
-                  )}
-                </Menu.Item>
-              ) : (
+              {!props.item.isBackup ? (
                 <>
                   <Menu.Item>
                     {({ active }) => (
                       <div
-                        onClick={onCopyLinkButtonClicked}
+                        onClick={onCreateFolderButtonClicked}
+                        className={`${
+                          active && 'bg-gray-5'
+                        } flex cursor-pointer items-center py-2 px-3 text-gray-80 hover:bg-gray-5`}
+                      >
+                        <FolderSimplePlus size={20} />
+                        <p className="ml-3">{i18n.get('actions.upload.folder')}</p>
+                      </div>
+                    )}
+                  </Menu.Item>
+                  <div className="my-0.5 mx-3 border-t border-gray-10" />
+                  {!isBreadcrumbItemShared ? (
+                    <Menu.Item>
+                      {({ active }) => (
+                        <div
+                          onClick={onCreateLinkButtonClicked}
+                          className={`${
+                            active && 'bg-gray-5'
+                          } flex cursor-pointer items-center py-2 px-3 text-gray-80 hover:bg-gray-5`}
+                        >
+                          <Link size={20} />
+                          <p className="ml-3">Get Link</p>
```

Comment:
Also in this file, check all texts that it need it and add the translation as I asked before

Reply:
added